src/main/java/com/jungdam/common/dto/ResponseMessage.java

```diff
@@ -9,7 +9,8 @@
     MEMBER_LOGIN_SUCCESS(HttpStatus.OK, "로그인 성공"),
     TOKEN_REFRESH_SUCCESS(HttpStatus.OK, "토큰 리프레시 성공"),
     IMAGE_UPLOAD_SUCCESS(HttpStatus.CREATED, "이미지 업로드 성공"),
-    DIARY_CREATE_SUCCESS(HttpStatus.CREATED, "일기 생성 성공");
+    DIARY_CREATE_SUCCESS(HttpStatus.CREATED, "일기 생성 성공"),
+    DIARY_READ_SUCCESS(HttpStatus.OK, "일기 조회 성공");
```

Comment:
데이비가 제안한 방식으로 아래처럼 바꿔주세요~
```suggestion
    DIARY_READ_SUCCESS(HttpStatus.OK, "일기 조회 성공"),
    ;
```

Reply:
앗! 넵넵!!!!